src/utils/validations.js

```diff
@@ -84,6 +84,14 @@ class ValidationAndCasting {
     return this;
   }
 
+  isConstrainedToSize(maxLength, errorMsg = `Object is to large`) {
```

Comment:
```suggestion
  isConstrainedToSize(maxLength, errorMsg = `Object is too large.`) {
```

Reply:
We usually do not add dot at the end of an error message.
```suggestion
  isConstrainedToSize(maxLength, errorMsg = `Object is too large`) {
```